### PR TITLE
Update Dev with previous release notes etc from release branch.

### PR DESCRIPTION
**Pull request by adewaleo**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/setup.py [hunk 1]
```diff
@@ -52,62 +52,62 @@
...
+    'azure-cli-search==0.1.1',
+    'azure-cli-security==0.1.0',
+    'azure-cli-servicebus==0.3.3',
+    'azure-cli-servicefabric==0.1.13',
+    'azure-cli-signalr==1.0.0',
+    'azure-cli-sql==2.1.9',
+    'azure-cli-sqlvm==0.1.0',
+    'azure-cli-storage==2.3.1',
+    'azure-cli-telemetry==1.0.1',
+    'azure-cli-vm==2.2.16'
```
Comment (tjprescott) [inline src/azure-cli/setup.py]: You have to unpin the versions in the dev branch or dev installation will break.

File: src/command_modules/azure-cli-resource/HISTORY.rst [hunk 2]
```diff
@@ -3,15 +3,18 @@
 Release History
 ===============
 
-* `deployment create`: Fix issue where type field was case-sensitive.
-
-2.1.10
+2.1.11
 ++++++
+* `deployment create`: Fix issue where type field was case-sensitive.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-resource/HISTORY.rst]: This looks wrong. The new line should be 2.1.11 but the following 4 lines should be part of 2.1.10...

Comment (adewaleo) [reply src/command_modules/azure-cli-resource/HISTORY.rst]: @tjprescott this is from the previous release. Looks good when I merged locally and compared to pypi.

**Comment by marstr** [comment]
Closing in favor of #8725

**Pull request closed without merging**